src/main/kotlin/com/geekbeast/rhizome/jobs/ResumeJobsInitializationTask.kt

```diff
@@ -0,0 +1,27 @@
+package com.geekbeast.rhizome.jobs
+
+import com.openlattice.tasks.HazelcastInitializationTask
+import java.util.*
+
+const val RESUME_JOBS_TASK = "_rhizome_resume_jobs_task_"
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+class ResumeJobsInitializationTask : HazelcastInitializationTask<ResumeJobDependencies> {
+    override fun getInitialDelay(): Long = 0
+
+    override fun initialize(dependencies: ResumeJobDependencies) {
+        dependencies
+                .jobService.getJobs(EnumSet.of(JobStatus.RUNNING, JobStatus.PENDING))
+                .forEach { (id, job) ->
+                    dependencies.jobService.submitJob(job)
```

Comment:
doesn't `submitJob` require a status of PENDING?

Reply:
Should be resume jobs.